ovos_utils/events.py

```diff
@@ -1,9 +1,81 @@
-from ovos_utils.log import LOG
-from ovos_utils.messagebus import Message, FakeBus
 import time
 from datetime import datetime, timedelta
 from inspect import signature
 
+from ovos_utils.intents.intent_service_interface import to_alnum
+from ovos_utils.log import LOG
+from ovos_utils.messagebus import Message, FakeBus
+
+
+def unmunge_message(message, skill_id):
+    """Restore message keywords by removing the Letterified skill ID.
+    Args:
+        message (Message): Intent result message
+        skill_id (str): skill identifier
+    Returns:
+        Message without clear keywords
+    """
+    if isinstance(message, Message) and isinstance(message.data, dict):
+        skill_id = to_alnum(skill_id)
+        for key in list(message.data.keys()):
+            if key.startswith(skill_id):
+                # replace the munged key with the real one
+                new_key = key[len(skill_id):]
+                message.data[new_key] = message.data.pop(key)
+
+    return message
+
+
+def get_handler_name(handler):
+    """Name (including class if available) of handler function.
```

Comment:
If I can pass it a function, don't I already have the function's name?

Reply:
used in intent decorators to create intent name for example, you have the handler but no clue what it is